transformer_engine/pytorch/module/linear.py

```diff
@@ -206,8 +224,8 @@ def forward(
                     if isinstance(inputmat_total, Float8Tensor)
                     else inputmat_total
                 ),
-                fp8_meta["scaling_fwd"].scale_inv,
-                tex.FP8FwdTensors.GEMM1_INPUT,
+                inputmat_scale_inv,
+                0,
```

Comment:
Why not just remove this item?

Reply:
Mostly to keep the API backward-compatible. `LayerNormMLP` is still storing scale-invs in the `fp8_meta`.

Reply:
Not sure I follow - this particular call is from internal autograd function, so we should be able to change its API.

Reply:
`fp8_gemm` is used differently in `Linear` and `LayerNormMLP`: `Linear` constructs a new scale-inv tensor, `LayerNormMLP` still uses the `fp8_meta`'s scale-inv and requires an offset. Changing the API for `fp8_gemm` would also require making changes in `te.Sequential`.